test.js

```diff
@@ -15,6 +15,51 @@ describe('timestring', function() {
     expect(timestring('1y')).to.equal(29030400);
   });
 
+  it('can parse different unit identifiers', function() {
+    var unitMap = {
```

Comment:
I had to duplicate the map since it isn't exported, but I think violating DRY is ok here since looping over the unit types really reduces the amount of code. Happy to change how I wrote this test if you disagree though!